torch/distributed/distributed_c10d.py

```diff
@@ -3234,3 +3235,41 @@ def new_subgroups_by_enumeration(
                 logger.info("Rank {} is assigned to subgroup {}".format(rank, ranks))
 
     return cur_subgroup, subgroups
+
+def broadcast_coalesced(
+    tensors: [torch.Tensor],
+    buffer_size: int,
```

Comment:
Should we provide a reasonable default so users don't have to pick this every time? i.e. maybe the DDP bucket size would be reasonable?